analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -318,11 +322,27 @@ public void onActivityStarted(Activity activity) {
           @Override
           public void onActivityResumed(Activity activity) {
             runOnMainThread(IntegrationOperation.onActivityResumed(activity));
+            if (shutdown) {
+              return;
+            }
+            track(
+                "Application Opened",
+                new Properties()
+                    .putValue("version", currentVersion)
+                    .putValue("build", currentBuild));
           }
 
           @Override
           public void onActivityPaused(Activity activity) {
             runOnMainThread(IntegrationOperation.onActivityPaused(activity));
+            if (shutdown) {
+              return;
+            }
+            track(
```

Comment:
Same as https://github.com/segmentio/analytics-ios/pull/831#discussion_r304657458 - I don't believe any properties should be collected here, as per https://segment.com/docs/spec/mobile/#application-backgrounded.